backend/src/main/java/com/bakdata/conquery/resources/api/ConceptsProcessor.java

```diff
@@ -163,40 +189,58 @@ public AutoCompleteResult autocompleteTextFilter(SelectFilter<?> filter, Optiona
 		final int pageNumber = pageNumberOpt.orElse(0);
 		final int itemsPerPage = itemsPerPageOpt.orElse(50);
 
-		final String text = maybeText.orElse("");
-
 		Preconditions.checkArgument(pageNumber >= 0, "Page number must be 0 or a positive integer.");
 		Preconditions.checkArgument(itemsPerPage > 1, "Must at least have one item per page.");
 
+		log.debug("Searching for for  `{}` in `{}`. (Page = {}, Items = {})", maybeText, filter.getId(), pageNumber, itemsPerPage);
```

Comment:
Würde ich auf Trace stellen

Reply:
> Würde ich auf Trace stellen

Ich hätte gerne zumindest ein bisschen Logging um die Suche herum. Auch um zu verstehen wie das ganze verwendet wird. Hier vmtl zu verbos, schiebe es mal in die LoadingCaches

Reply:
Sonst stelle in der Config ein, dass diese ein klasse auf niedrigerem Level gelogget werden soll